app/views/evaluator_submission_assignments/_submission_row.html.erb

```diff
@@ -1,25 +1,32 @@
-<tr data-evaluator-id="<%= evaluator.id %>" data-evaluator-type="user">
+<tr data-evaluator-id="<%= evaluator.id %>" data-evaluator-type="user" class="border-y-2px border-primary" style="height: 4rem;">
```

Comment:
I'm not sure why these styles were added? removing them doesn't seem to affect anything. please remove if they're not doing anything. maybe it was added before the table styling updates on friday.

Reply:
```suggestion
<tr data-evaluator-id="<%= evaluator.id %>" data-evaluator-type="user">
```